content/post/000-ipfs-in-web-browsers.md

```diff
@@ -0,0 +1,90 @@
+
+But we're making headway. Today, we'd like to share some collaborations the IPFS project has had in the works for a while, which bring us a few steps closer to making unmediated access to information *just work*... by solving that "last mile" problem and *integrating IPFS directly into web browsers*.
+
+## Stages
+
+(INSERT STAGES DIAGRAM)
+
+The path to a truly decentralized web is a long one. For over 30 years the browser has been a [*client*](https://en.wikipedia.org/wiki/Client_(computing)) - but a foundational concept in P2P systems is that a participant is both a client *and a server*. Web browser vendors and web standards bodies have not designed for this architectural shift, so we're breaking it down into steps.
+
+From the beginning, IPFS had an HTTP gateway. The gateway lets HTTP clients like web browsers publish to and read from the IPFS network. Now there are [lots of public HTTP gateways to the IPFS network](https://ipfs.github.io/public-gateway-checker/), and ipfs.io serves XXX requests per day.
+
+Initially in our browser work, we're aiming to ease access to the IPFS HTTP gateways - creating a bridge between the billions of people on the centralized web and content on the distributed web. We expect some experimentation in the next phase - adjusting node behavior based on browser APIs, device capabilities, bandwidth availability and power consumption - while we grow adoption with our browser partners as we learn and build common understanding at standards bodies. Ultimately, we aim to have multiple browsers implement fully native (and interoperable!) IPFS support.
+
+As part of this progression to native IPFS support, we've had ongoing collaborations with Firefox, Brave, Opera, and other browsers. Read on to hear the latest status and learn about our progress so far.
+
+## Brave
+
+Brave has tended to live up to its name since the beginning - riling up the browser, publisher and crypto worlds. [Since 2018](https://github.com/brave/brave-browser/issues/819) we've been working with Brave on IPFS support in some form, and the results of that work are now emerging.
+
+Today you can open [brave://settings/extensions](brave://settings/extensions) in Brave and see a toggle for enabling IPFS Companion.
+
+![Enabling IPFS Companion in Brave](https://file.globalupload.io/ZVxjmPZ48E.gif)
+
+Removing this install step *really matters* for adoption. While technical users are familiar with browser extensions, it's less comfortable for non-technical users, and Brave users have the additional overhead of going to the *Chrome* web app store to find and install them.
+
+In the future, when you [load a URL beginning with "ipfs://"](https://github.com/brave/brave-browser/issues/3045), Brave will prompt you to enable the extension, providing a smooth discovery and onboarding experience.
```

Comment:
Feels like a good place to mention we also think about address bar UX (implies native protocol handler API and other technical stuff without going technical):
```suggestion
In the future, when you [load a URL beginning with "ipfs://"](https://github.com/brave/brave-browser/issues/3045), Brave will prompt you to enable the extension, providing a smooth discovery and onboarding experience. And when you are browsing IPFS resource, the [address bar will reflect that](https://github.com/brave/brave-browser/issues/5218).
```